src/gui/mainwindow.cpp

```diff
@@ -2003,7 +2003,11 @@ void MainWindow::installPython()
 {
     setCursor(QCursor(Qt::WaitCursor));
     // Download python
-    const QString installerURL = "https://www.python.org/ftp/python/3.6.6/python-3.6.6.exe";
+#ifdef QBT_APP_64BIT
```

Comment:
This probably isn't correct when user is running 32-bit qbt on x64 Windows. To fully detect this situation is to utilize [`IsWow64Process()`](https://docs.microsoft.com/en-us/windows/win32/api/wow64apiset/nf-wow64apiset-iswow64process) and `QBT_APP_64BIT` together, however I regard that is a bit too much so I kept the current for simplicity.